### PR TITLE
MRR movements adjustments

### PR DESCRIPTION
**Pull request by AleckVictor**
Several changes where made aiming to make mrr movements view more acurate:

- Churn always has ending mrr equal to zero.
- Churn event happens one month after previous movement.
- If there is no mrr on the current month subscription, a churn event is registered.
- Removing negative MRR from calculation.

### CODE REVIEW

File: models/stripe__subscription_items_mrr.sql [hunk 1]
```diff
@@ -43,5 +46,5 @@ left join price
     on subscription_item.plan_id = price.id
 left join product
     on price.product_id = product.id
-where 
-    subscription.status IN ('active', 'past_due')
+--where 
+    --subscription.status IN ('active', 'past_due')
```
Comment (fmluizao) [inline models/stripe__subscription_items_mrr.sql:50]: Why keep commented code? That's why we have git history :smile:

**Review by fmluizao — APPROVED** [review]
LGTM, great job Aleck, just some minor notes

:1st_place_medal:

File: models/mart/mrr_movements_monthly.sql [hunk 2]
```diff
@@ -26,21 +27,27 @@ churn as (
 		date_trunc('month', mrr_day)::date as mrr_month,
 		event_type as churn_event,
 		stripe_account
-	from  {{ref('mrr_movements')}}
+	from  "defaultdb"."dbt_aoliveira_data_marts"."mrr_movements"
```
[the remark this answers is not in the document]
Comment (AleckVictor) [reply models/mart/mrr_movements_monthly.sql:30]: No! I will adjust, thanks!

File: models/stripe__subscription_items_mrr.sql (hunk 1 above)
Comment (AleckVictor) [reply models/stripe__subscription_items_mrr.sql:50]: No, I just forgot to remove it, thanks!

**Pull request merged**